spinegeneric/cli/generate_figure.py

```diff
@@ -785,6 +897,122 @@ def format_number(number):
     logger.info('Created: ' + fname_fig)
 
 
+def generate_figure_t1_t2_plotly(df,csa_t1,csa_t2):
```

Comment:
please check PEP8 everwhere
```suggestion
def generate_figure_t1_t2_plotly(df, csa_t1, csa_t2):
```

Reply:
@noemieroberge some of my comments are not taken into account, e.g. this one